CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,248 @@
+
+* Use all platforms for appropiate class reasons 
+* Follow the instructions while using the platforms
+* Keeping "up-to-date" with platforms may require seeking additional extra-curricular knowledge
+  * For example, using Markdown in Discord
+* Respect others' thoughts, time, and contributions to either discussion or code product
+* Check course platforms often -- at least twice a day
+  * The best times to review are often the beginning and ends of a day
+    * i.e.: 8:00 (beginning), 18:00 (end)
+
+In addition, I suggest that you at least download and install the Discord application, rather than use the web interface. This is not required, but will help greatly with your ability to keep up with notifications.
+
+#### Using office hours
+
+When using office hours:
+
+* Come in prepared with questions, a topic, or a clear objective
+  * To achieve this, please add notes about expected topic(s) to a Google Calendar slot reservation (for instructor calendar)
+* Be prepared to recieve feedback, understanding that typically this means completing revisions on what ever you are working on
+* Respect the time of other citizens (i.e. be mannerly and quick)
+* Have all assignments ready to discuss with the Professor
+* Not rely only on office hours (i.e. work should be attempted first)
+ 
+#### Attendance
+
+Especially in precarious times like those in which we find ourselves, I _cannot_ and _will not_ require you to be physically present for class sessions. However, if you feel well and comfortable, I (and your peers) ask that you attend class.
+
+As many of your note, this isn't something that I provide grading criteria for. I don't really have an interest in directly grading attendance. However, _not attending_ traditionally shows up in final products and, thereby, grades. It's important to remember that attendance and participation are closely related. Not doing either severely restricts your ability to succeed in the course.
+
+It is, however, also possible that a citizen many not feel well enough to attend even a portion of a remote course. In these situations, citizens must:
+
+* Use their neighborhood as a source of knowledge for given content missed
+* Failing the neighborhood's ability to "crowd-source" an answer, citizens should [use office hours](#Using-office-hours) to make up the gap in understanding
+  * These hours _must be_ with the Mayor, and not a TL if intending to "catch up" on missed work
+  * citizens who have missed a class may use TL office hour time to if they feel that they understand the content, but have questions about approach or methods       
+
+### Neighborhood contributions
+
+This course works due to the trust the Mayor places in each neighborhood becoming a collaborative home for all citizens. However, there is a fine line between collaboration and academic dishonesty. Here, we define that that line is and what general consequences (positive or negative) are under it.
```

Comment:
Should each neighborhood elect a spokesperson or representative or just act communally?

Reply:
It would be interesting to allow each neighborhood to discover and decide how to allocate their collaborative tasks. This could include electing representatives or spokespersons, or they could rotate responsibilities for a favorable and fair outcome.